lib/byron/test/data/cardano-node/node.config

```diff
@@ -0,0 +1,73 @@
+# vim: set filetype=yaml:
```

Comment:
this one is probably not needed ;-)

Reply:
And the one below is for Emacs users. Because obviously, there are only two text editors out there :no_mouth: